api/ccapi/include/model.h

```diff
@@ -28,6 +28,8 @@
 #include <layer.h>
 #include <optimizer.h>
 
+#include <tensor_dim.h>
```

Comment:
You will have to package tensor_dim.h with ccapi API, and move the header `tensor_dim.h` to ccapi location.
Then update the namespace to ml::train::TensorDim, and create an alias nntrainer::TensorDim.

Reply:
Update tizen, debian and android packaging.

Reply:
Are we going to open tensor_dim to public?

Reply:
with this API design, TensorDim will have to be opened? or maybe the return type can be vector<int> but it will not completely represent TensorDim.

Reply:
I mean this model.h is opened to user.. if tensor_dim.h is included this header.. then it means open to public. doesn't it?

Reply:
Yes, you are right.